ImportError: cannot import name '_register_test_and_source_path_and_return_test_dir'

This issue might make sense here also, at least as an FYI?
https://github.com/AlexeySanko/pybuilder_pytest/issues

We are hitting it too:

```
[2020-04-21T01:06:41.491Z] app_1  | ------------------------------------------------------------

[2020-04-21T01:06:41.491Z] app_1  | BUILD FAILED - Missing plugin plugin 'pypi:pybuilder_pytest', module 'pybuilder_pytest': Traceback (most recent call last):

[2020-04-21T01:06:41.491Z] app_1  |   File "/usr/local/lib/python3.6/site-packages/pybuilder/pluginloader.py", line 175, in _load_plugin

[2020-04-21T01:06:41.491Z] app_1  |     __import__(plugin_module_name)

[2020-04-21T01:06:41.491Z] app_1  |   File "/usr/local/lib/python3.6/site-packages/pybuilder/reactor.py", line 532, in __import_with_plugins

[2020-04-21T01:06:41.491Z] app_1  |     return self._old_import(*args, **kwargs)

[2020-04-21T01:06:41.491Z] app_1  |   File "/opt/redox-local-k8s/.pybuilder/plugins/cpython-3.6.10.final.0/lib/python3.6/site-packages/pybuilder_pytest/__init__.py", line 22, in <module>

[2020-04-21T01:06:41.491Z] app_1  |     from pybuilder.plugins.python.unittest_plugin \

[2020-04-21T01:06:41.491Z] app_1  | ImportError: cannot import name '_register_test_and_source_path_and_return_test_dir'
```

Downgrading to 0.12.1 works.

--- a/src/main/python/pybuilder/plugins/python/unittest_plugin.py
+++ b/src/main/python/pybuilder/plugins/python/unittest_plugin.py
@@ -233,6 +233,15 @@ def _instrument_result(logger, result):
     return result
 
 
+def _register_test_and_source_path_and_return_test_dir(project, system_path, execution_prefix):
+    """This function is deprecated and will be removed and should not be used by any new code"""
+    test_dir = project.expand_path("$dir_source_%s_python" % execution_prefix)
+    system_path.insert(0, test_dir)
+    system_path.insert(0, project.expand_path("$dir_source_main_python"))
+
+    return test_dir
+
+
 def write_report(name, project, logger, result, console_out):
     project.write_report("%s" % name, console_out)
 

